File: java/core/src/main/java/com/github/fernthedev/lightchat/core/packets/latency/PingReceive.java
package com.github.fernthedev.lightchat.core.packets.latency;

import com.github.fernthedev.lightchat.core.packets.Packet;
import com.github.fernthedev.lightchat.core.packets.PacketInfo;

@PacketInfo(name = "PING_RECEIVE")
public class PingReceive extends Packet implements LatencyPacket{

}

File: java/core/src/main/java/com/github/fernthedev/lightchat/core/packets/latency/PingReceive.kt
package com.github.fernthedev.lightchat.core.packets.latency

import com.github.fernthedev.lightchat.core.packets.PacketJSON
import com.github.fernthedev.lightchat.core.packets.PacketInfo

@PacketInfo(name = "PING_RECEIVE")
class PingReceive : PacketJSON(), LatencyPacket